src/cli/commands/files/cat.js

```diff
@@ -36,7 +36,7 @@ module.exports = Command.extend({
           throw (err)
         }
         res.on('data', (data) => {
-          data.stream.pipe(process.stdout)
+          data.content.pipe(process.stdout)
         })
```

Comment:
Yup, changes to make cat return just a stream are in <a href="https://github.com/ipfs/js-ipfs/pull/256">PR 256</a>